common/app/layout/Sublink.scala

```diff
@@ -135,18 +135,33 @@ object SnapStuff {
       case link: LinkSnap => link.enriched.flatMap(_.embedHtml)
       case _ => None
     }
+
+    val embedCss = faciaContent match {
+      case curated: CuratedContent => curated.enriched.flatMap(_.embedCss)
+      case link: LinkSnap => link.enriched.flatMap(_.embedCss)
+      case _ => None
+    }
+
+    val embedJs = faciaContent match {
+      case curated: CuratedContent => curated.enriched.flatMap(_.embedJs)
+      case link: LinkSnap => link.enriched.flatMap(_.embedJs)
+      case _ => None
+    }
+
     faciaContent.properties.embedType match {
-      case Some("latest") => Some(SnapStuff(snapData, faciaContent.properties.embedCss, FrontendLatestSnap, embedHtml))
-      case Some("link") => Some(SnapStuff(snapData, faciaContent.properties.embedCss, FrontendLinkSnap, embedHtml))
-      case Some(_) => Some(SnapStuff(snapData, faciaContent.properties.embedCss, FrontendOtherSnap, embedHtml))
+      case Some("latest") => Some(SnapStuff(snapData, faciaContent.properties.embedCss, FrontendLatestSnap, embedHtml, embedCss, embedJs))
+      case Some("link") => Some(SnapStuff(snapData, faciaContent.properties.embedCss, FrontendLinkSnap, embedHtml, embedCss, embedJs))
+      case Some(_) => Some(SnapStuff(snapData, faciaContent.properties.embedCss, FrontendOtherSnap, embedHtml, embedCss, embedJs))
```

Comment:
Reviewing this now, I wonder if I only need to be including `embedCss` and `embedJs` if the `embedType` is `interactive`. I don't think those would be set in any other context. I think the same is probably true of the `faciaContent match` lines above (`embedJs` and `embedCss` probably aren't set in all contexts.